Packs/CVESearch/Integrations/CVESearchV2/CVESearchV2_test.py

```diff
@@ -1,31 +1,34 @@
 import json
 import os
-from CVESearchV2 import cve_command, valid_cve_id_format, Client
-from CommonServerPython import DemistoException, argToList
+from pathlib import Path
+from CVESearchV2 import cve_command, valid_cve_id_format, Client, generate_indicator, parse_cpe
+from CommonServerPython import DemistoException, argToList, EntityRelationship
 import pytest
+import re
 
 BASE_URL = 'https://cve.circl.lu/api/'
 
 
 def test_wrong_path():
     bad_url = 'https://cve.bad_url'
     client = Client(base_url=bad_url)
-    try:
+
+    with pytest.raises(DemistoException) as excinfo:
         cve_command(client, {"cve_id": 'cve-2000-1234'})
-        assert False, 'Bad url- Exception should by raised'
-    except DemistoException as err:
-        expected_exception_message = 'Verify that the server URL parameter is correct'
-        assert expected_exception_message in str(err), 'Bad error response when bad url is given'
+
+    expected_exception_message = 'Verify that the server URL parameter is correct'
+    assert expected_exception_message in str(excinfo.value), 'Bad error response when bad URL is given'
 
 
 def test_bad_cve_id():
```

Comment:
Why not use @pytest.mark.parametrize and control the url and cve_id to test the exception message?
Those 2 tests are almost the same.

Reply:
One checks for URL validity and one for CVE structure, are you sure we want to combine it? Seems like bad practice